### ISSUE TITLE
Computed panel is not working properly

### ISSUE THREAD

**Issue by pinta83** [firebug/firebug#8051]
I think the problem is with Firefox version 49 altough I'm not 100% sure...

The problem is that computed panel is either empty or is missing 80% properties....

I have the same issue on Ubuntu 14.04 (Firefox 49.0) and Win7 x64 (Firefox 49.1) in combination with Firebug 2.0.17....

Here are some screenshots from my Ubuntu workstation....

![firebug2](https://cloud.githubusercontent.com/assets/2417948/19339624/cf95791a-9124-11e6-88e9-f90267203183.png)
![firebug](https://cloud.githubusercontent.com/assets/2417948/19339625/cf99a210-9124-11e6-9c46-ae77969c25f5.png)

**Comment by pinta83** [comment]
Still the same problem with Firebug 2.0.18

**Comment by idwasga** [comment]
I've got the same issue with Firebug 2.0.18 on Firefox 49.0.1, Windows 10. Strangely, when I click away from the Computed tab onto another such as Style or Layout, then back to the Computed tab, it starts to show the missing values, but usually only some to begin with - I have to click away from and back to the Computed tab several times for all the missing values to be shown. Then, however, if I close and re-open Firebug, all the tabs have completely disappeared with no obvious way to get them back except to restart Firefox.

**Comment by pinta83** [comment]
Yeah it start showing some computed style but not all if u navigate to another tab, and then return... For example, you will never get the font-size property (if it's not explicitly defined on the element you are viewing at the moment)
